cpython/Setup.local

```diff
@@ -35,6 +35,9 @@ _sha3 _sha3/sha3module.c
 _md5 md5module.c
 _blake2 _blake2/blake2module.c _blake2/blake2b_impl.c ../../host/Python-3.7.0/Modules/_blake2/blake2s_impl.c
 
+_sqlite3 _sqlite/cache.c _sqlite/connection.c _sqlite/cursor.c _sqlite/microprotocols.c _sqlite/module.c _sqlite/prepare_protocol.c _sqlite/row.c _sqlite/statement.c _sqlite/util.c -I/src/cpython/build/sqlite-snapshot-201903021525 -L-I/src/cpython/build/sqlite-snapshot-201903021525 -lsqlite3
```

Comment:
There are some absolute paths in here (starting with `/src`) and the puzzling `-L-I/src...`.  Also, as mentioned above, let's try statically linking to sqlite by adding `libsqlite3.la` here instead of `-lsqlite3`.

Reply:
Hmmm I wonder if the `-L-I` junk, which was clearly a typo on my part, is why I ended up linking to `sqlite3` in the top-level `Makefile`. Going to correct this and do some testing...